### PR TITLE
Extension dev enable module path correction

### PR DESCRIPTION
**Pull request by ansari-krish**
## This PR is a:

- [ ] New topic
- [ ] Content update
- [ ] Content fix or rewrite
- [X] Bug fix or improvement

## Summary

When this pull request is merged, it will...
Admin path for enable module list is displaying wrong.I have changed it to the correct path.

## Additional information

List all affected URLs 

https://devdocs.magento.com/guides/v2.3/extension-dev-guide/build/enable-module.html

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=3914) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=3914) before we can accept your contribution.<br/>

**Comment by magento-cicd2** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.1/extension-dev-guide/build/enable-module.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ After you have built the component and are ready to enable it in your Magento en
 
    Where `Component_Name` is the name of the component you are enabling.
 
-1. Check under `Stores->Configuration->Advanced->Advanced` that the component is present.
+1. Check under `System->Web Setup Wizard->Module Manager` that the component is present.
```
Comment (osrecio) [inline guides/v2.1/extension-dev-guide/build/enable-module.md]: I would add the `Tools` section
```suggestion
1. Check under `System->Tools->Web Setup Wizard->Module Manager` that the component is present.
```

Comment (ansari-krish) [reply guides/v2.1/extension-dev-guide/build/enable-module.md]: Please review now, Changes are done.

**Comment by hguthrie** [comment]
running tests

**Pull request merged**